# Static joint limits parameters in degrees for a1
joint0_min_limit: -46
joint0_max_limit: 46

joint1_min_limit: -65
joint1_max_limit: 245

joint2_min_limit: -160
joint2_max_limit: -60

safety_spread: 0.99